src/tests/ftest/util/agent_utils.py

```diff
@@ -97,6 +97,45 @@ def get_params(self, test):
         # Run daos_agent with test variant specific log file names if specified
         self.yaml.update_log_file(getattr(test, "agent_log"))
 
+    def get_sub_command_class(self):
+        """Get the daos_agent sub command object based on the sub-command."""
+        if self.sub_command.value == "dump-attachinfo":
+            self.sub_command_class = self.DumpAttachInfoSubCommand()
+        else:
+            self.sub_command_class = None
+
+    class DumpAttachInfoSubCommand(CommandWithSubCommand):
+        """Defines an object for the daos_agent dump-attachinfo sub command."""
+
+        def __init__(self):
+            """Create a daos_agent dump-attachinfo subcommand object."""
+            super(DaosAgentCommand.DumpAttachInfoSubCommand, self).__init__(
+                "/run/daos_agent/dump-attachinfo/*", "dump-attachinfo")
+
+            self.output = FormattedParameter("--output {}", None)
+
+    def dump_attachinfo(self, path="uri.txt"):
+        """Write CaRT attachinfo file
+
+        Args:
+            path (str): Path to attachinfo file.
```

Comment:
Isn't this actually the full path to an output file?

Reply:
updated description to "Path to attachinfo output file"